EIPS/eip-7213.md

```diff
@@ -0,0 +1,328 @@
+---
+eip: 7213
```

Comment:
```suggestion
eip: 7432
```
Assigning EIP number as the PR number.  #7213 is an unrelated PR.
Please also change the filename

Reply:
Done, Thank you! @abcoathup